vimeo-networking/src/main/java/com/vimeo/networking/model/playback/Play.java

```diff
@@ -0,0 +1,206 @@
+
+/**
+ * Playback
+ * <p/>
+ * Created by zetterstromk on 4/25/16.
+ */
+public class Play implements Serializable {
+
+    private static final long serialVersionUID = -7429617944240759711L;
+
+    public enum Status {
+        // if not done transcoding
+        @SerializedName("unavailable")
+        UNAVAILABLE("unavailable"),
+        // can be played. the video is marked as done transcoding
+        @SerializedName("playable")
+        PLAYABLE("playable"),
+        // can be purchased. is an on demand video that the user has not purchased
+        @SerializedName("purchase_required")
+        PURCHASE_REQUIRED("purchase_required"),
+        // your region can not play or purchase this
+        @SerializedName("restricted")
+        RESTRICTED("restricted");
+
+        private String string;
+
+        Status(String string) {
+            this.string = string;
+        }
+
+        @Override
+        public String toString() {
+            return this.string;
+        }
+    }
+
+    @Nullable
+    @SerializedName("embed")
+    private Embed mEmbed;
+    @Nullable
+    @SerializedName("hls")
+    private VideoFile mHls;
+    @Nullable
+    @SerializedName("dash")
+    private VideoFile mDash;
+    @Nullable
+    @SerializedName("progressive")
+    private ArrayList<VideoFile> mProgressive;
+    @Nullable
+    @SerializedName("progress")
+    private PlayProgress mProgress;
+    @Nullable
+    @SerializedName("status")
+    private Status mStatus;
+
+    @Nullable
+    public Embed getEmbed() {
+        return mEmbed;
+    }
+
+    public void setEmbed(@Nullable Embed embed) {
+        this.mEmbed = embed;
+    }
+
+    @Nullable
+    public VideoFile getHlsVideoFile() {
+        return mHls;
+    }
+
+    @Nullable
+    public VideoFile getDashVideoFile() {
+        return mDash;
+    }
+
+    @Nullable
+    public ArrayList<VideoFile> getProgressiveVideoFiles() {
+        return mProgressive;
+    }
+
+    @Nullable
+    public PlayProgress getProgress() {
+        return mProgress;
+    }
+
+    public void setProgress(@Nullable PlayProgress progress) {
+        this.mProgress = progress;
+    }
+
+    @Nullable
+    public Status getStatus() {
+        return mStatus;
+    }
+
+    // -----------------------------------------------------------------------------------------------------
+    // Helpers
+    // -----------------------------------------------------------------------------------------------------
+    // <editor-fold desc="Helpers">
+    public int getFileCount() {
+        int count = 0;
+        if (getHlsVideoFile() != null) {
+            count++;
+        }
+        if (getDashVideoFile() != null) {
+            count++;
+        }
+        if (getProgressiveVideoFiles() != null) {
+            count += getProgressiveVideoFiles().size();
+        }
+        return count;
+    }
+
+
+    @Nullable
+    public String getLoadLoggingUrl() {
+        if (getHlsVideoFile() != null && getHlsVideoFile().getLog() != null &&
```

Comment:
This section of code could be terser if you just accessed your members directly. (e.g. mHls instead of getHlsVideoFile)